ee-feature-pack/ee-10-api/src/main/resources/modules/system/layers/base/jakarta/json/bind/api/main/module.xml

```diff
@@ -27,6 +27,8 @@
     </resources>
 
     <dependencies>
-        <module name="javax.json.api"/>
+        <module name="jakarta.json.api"/>
```

Comment:
Please don't make this kind of change in unrelated PRs. It's just going to lead to randomness in how we handle this.

Reply:
Oops, I didn't mean to commit that. FWIW this does seem like something we should change at some point as we have to locate and read two files to load modules this way.

Reply:
Yes. There have been conversations about it; we'll do it at some point. Probably soon.

https://wildfly.zulipchat.com/#narrow/stream/174184-wildfly-developers/topic/Modules.20javax.20dependencies